code/datums/ai/basic_mobs/basic_subtrees/speech_subtree.dm

```diff
@@ -32,7 +36,10 @@
 		else if(random_number_in_range <= (audible_emotes_length + non_audible_emotes_length))
 			controller.queue_behavior(/datum/ai_behavior/perform_emote, pick(emote_see))
 		else
-			controller.queue_behavior(/datum/ai_behavior/perform_speech, pick(speak))
+			if(sound.len)
```

Comment:
`pick(null)` and `pick(list())` are still going to give you null, so this check doesn't need to exist

Reply:
![image](https://user-images.githubusercontent.com/66640614/233750708-2e9077fa-7970-4ba4-8461-434101570e99.png)

Ran into this a little when doing my last PR, but pick doesn't like empty lists.

Reply:
That's so silly, ok

Reply:
I think I would prefer this be `sounds.len > 0 ? pick(sounds) : null` just so that we keep it to one call site but I don't care one way or the other